src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,20 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo }) => {
+  const { completed, title, user } = todo;
+
+  return (
+    <article
+      className={`TodoInfo ${completed
+        ? 'TodoInfo--completed'
+        : ''}`}
+    >
```

Comment:
Or you can rewrite like this
```suggestion
      className={`TodoInfo ${completed && 'TodoInfo--completed'}`}
    >
```